zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -500,6 +501,10 @@ public void loadData() throws IOException, InterruptedException {
         takeSnapshot();
     }
 
+    public boolean compareDigest(TxnHeader header, Record txn, TxnDigest digest) {
```

Comment:
This method is not used in code base. Is this still needed?

Reply:
Good catch, this is the method we used in a later feature to check consistency during DIFF sync, I'll remove it here.